File: JAVA/03-function/evenNumberSum.java
public class evenNumberSum {
    static void sumEvenNumber() {
        int sum = 0;
        for (int i = 1; i <= 10; i++) {
            if (i % 2 == 0) {
                sum += i;
            }

        }
        System.out.println(sum);
    }

    public static void main(String[] args) {
        sumEvenNumber();
        
    }
    
}

File: INTERVIEW_PREPARATION/CODING_ROUNDING/evennumbersum.js
const nums = [2, 5, 8, 11, 14, 17, 20];
let sum = 0;
for (let i = 0; i < nums.length; i++){
    if (nums[i] % 2 === 0) {
        sum += nums[i];
        
    }
}
console.log(sum);

